lib/hammer_cli_katello.rb

```diff
@@ -43,6 +43,11 @@ def self.exception_handler_class
                                           'hammer_cli_katello/organization'
                                          )
 
+  HammerCLI::MainCommand.lazy_subcommand!("alternate-content-sources", _("Manipulate content sources"), # rubocop:disable LineLength
```

Comment:
```suggestion
  HammerCLI::MainCommand.lazy_subcommand!("alternate-content-sources", _("Manipulate alternate content sources"), # rubocop:disable LineLength
```
I'd keep 'alternate' in there because a normal content source could potentially mean something else.